console-subscriber/src/builder.rs

```diff
@@ -149,6 +159,279 @@ impl Builder {
+    ///
+    /// In addition to the [`TasksLayer`], which collects instrumentation data
+    /// consumed by the console, the default [`Subscriber`][sub] initialized by this
+    /// function also includes a [`tracing_subscriber::fmt`] layer, which logs
+    /// tracing spans and events to stdout. Which spans and events are logged will
+    /// be determined by the `RUST_LOG` environment variable.
+    ///
+    /// **Note**: this function sets the [default `tracing` subscriber][default]
+    /// for your application. If you need to add additional layers to this
+    /// subscriber, see [`build`].
+    ///
+    /// [default]: https://docs.rs/tracing/latest/tracing/dispatcher/index.html#setting-the-default-subscriber
+    /// [sub]: https://docs.rs/tracing/latest/tracing/trait.Subscriber.html
+    /// [`tracing_subscriber::fmt`]: https://docs.rs/tracing-subscriber/latest/tracing-subscriber/fmt/index.html
+    /// [`Server`]: crate::Server
+    ///
+    /// ## Configuration
+    ///
+    /// Tokio console subscriber is configured with sensible defaults for most
+    /// use cases. If you need to tune these parameters, several environmental
+    /// configuration variables are available:
+    ///
+    /// | **Environment Variable**            | **Purpose**                                                               | **Default Value** |
+    /// |-------------------------------------|---------------------------------------------------------------------------|-------------------|
+    /// | `TOKIO_CONSOLE_RETENTION_SECS`      | The number of seconds to accumulate completed tracing data                | 3600s (1h)        |
+    /// | `TOKIO_CONSOLE_BIND`                | A HOST:PORT description, such as `localhost:1234`                         | `127.0.0.1:6669`  |
+    /// | `TOKIO_CONSOLE_PUBLISH_INTERVAL_MS` | The number of milliseconds to wait between sending updates to the console | 1000ms (1s)       |
+    /// | `TOKIO_CONSOLE_RECORD_PATH`         | The file path to save a recording                                         | None              |
+    /// | `RUST_LOG`                          | Configures what events are logged events. See [`Targets`] for details.    | "error"           |
+    ///
+    /// ## Further customization
+    ///
+    /// To add additional layers or replace the format layer, replace
+    /// `console_subscriber::Builder::init` with:
+    ///
+    /// ```rust
+    /// use tracing_subscriber::prelude::*;
+    ///
+    /// let console_layer = console_subscriber::TasksLayer::builder().build_console_layer();
+    ///
+    /// tracing_subscriber::registry()
+    ///     .with(console_layer)
+    ///     .with(tracing_subscriber::fmt::layer())
+    /// //  .with(..potential additional layer..)
+    ///     .init();
+    /// ```
+    ///
+    /// [`Targets`]: https://docs.rs/tracing-subscriber/latest/tracing-subscriber/filter/struct.Targets.html
+    pub fn init(self) {
+        let fmt_filter = std::env::var("RUST_LOG")
```

Comment:
what do you think about making the filter environment variable a builder parameter? so, by default, it uses `RUST_LOG`, but there's a separate builder method to change the filter env var? something like:

```rust
let console = console_subscriber::Builder::new()
    .with_filter_env_variable("MYAPP_LOG")
    .build_console_layer();
```

Reply:
(it would be fine --- or perhaps preferable! --- to add that in a follow-up PR)